### PR TITLE
NI-6597: S3S2 data transfer issue with EMOL partner

### PR DESCRIPTION
**Pull request by vikas-kph**

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ import (
 	session "github.com/aws/aws-sdk-go/aws/session"
 	options "github.com/tempuslabs/s3s2/options"
 	client "github.com/aws/aws-sdk-go/aws/client"
+    retryer "github.com/tempuslabs/s3s2/retryer"
```
Comment (iostreamdoth) [inline utils/utils.go]: spaces dont align.

**Comment by iostreamdoth** [comment]
were you able to build it locally and test it ?

File: utils/utils.go (hunk 1 above)
Comment (vikas-kph) [reply utils/utils.go]: @iostreamdoth It is aligned now

**Comment by vikas-kph** [comment]
> were you able to build it locally and test it ?

@iostreamdoth yes I have built it locally and tested the custom retryer

**Review by iostreamdoth — APPROVED** [review]

**Pull request merged**